### ISSUE TITLE
Unity-Builder repo secrets incorrect

### ISSUE THREAD

**Issue by AndrewKahr** [game-ci/unity-builder#507]
**Bug description**

It seems that `UNITY_EMAIL`, `UNITY_PASSWORD`, and/or `UNITY_SERIAL` are not configured in the repo causing windows and mac build tests to fail

**How to reproduce**

Run the mac or windows build workflows

**Expected behavior**

Tests should run successfully

**Additional details**

**Comment by AndrewKahr** [comment]
@davidmfinol It seems the serial isn't assigned to whoever is logged into the unity account for CI on windows/mac now

**Comment by AndrewKahr** [comment]
@GabLeRoux I think this is actually more your area.

**Comment by GabLeRoux** [comment]
I recently changed the credentials in the repository as per @frostebite's request because of a small problem that occurred with github actions secrets.

![image](https://user-images.githubusercontent.com/1264761/220145347-66dce3a3-0506-4d0c-b5c6-07f016392669.png)

I don't exactly remember which credentials were there before. When updating it, I used a unity account that is dedicated to that project and has nothing else on (for security reasons). I wonder if the problem is more about the `UNITY_LICENSE` or the `UNITY_SERIAL`.

Actually, If I understand correctly, if we're using a `UNITY_SERIAL`, the logged in account should also be linked to that serial, I'll poke the maintainers and see with them what should be the setup for that repo 👍 Thanks for the ping, I'll update here asap.
